controller/replica_controller.go

```diff
@@ -294,6 +294,10 @@ func (rc *ReplicaController) getReadinessProbeFailureThreshold(r *longhorn.Repli
 	return int32(r.Spec.VolumeSize / replicaReadinessProbeMinimalRestoreRate / replicaReadinessProbePeriodSeconds)
 }
 
+func singleQuotes(static string) string {
```

Comment:
Got an issue?

Reply:
Yeah, the issue is the way we construct command line arguments with unescaped user input which happens to have hypens and confuses the CLI parser. Solution is to treat user input as shell unevaluated string.